### PR TITLE
switch input to textarea

### PR DESCRIPTION
**Pull request by asylves1**
# Description
image

Design Change:
- in equation from model change the input fields to text areas

Resolves #(issue)

### CODE REVIEW

**Review by YohannParis — APPROVED** [review]

File: packages/client/hmi-client/src/components/workflow/ops/model-from-equations/tera-model-from-equations-drilldown.vue [hunk 1]
```diff
@@ -131,10 +135,14 @@
 											</div>
 										</div>
 									</section>
-									<tera-input-text
+									<Textarea
 										v-if="selectedItem === equation.name"
 										v-model="equation.asset.text"
+										ref="equationTextarea"
```
[the remark this answers is not in the document]
Comment (asylves1) [reply packages/client/hmi-client/src/components/workflow/ops/model-from-equations/tera-model-from-equations-drilldown.vue:140]: removed

[the remark this answers is not in the document]
Comment (shawnyama) [reply packages/client/hmi-client/src/components/workflow/ops/model-from-equations/tera-model-from-equations-drilldown.vue:140]: the other textarea still has one

[the remark this answers is not in the document]
Comment (shawnyama) [reply packages/client/hmi-client/src/components/workflow/ops/model-from-equations/tera-model-from-equations-drilldown.vue:140]: the other textarea still has one

[the remark this answers is not in the document]
Comment (asylves1) [reply packages/client/hmi-client/src/components/workflow/ops/model-from-equations/tera-model-from-equations-drilldown.vue]: removed

**Review by mwdchang — APPROVED** [review]

**Pull request merged**